src/test/java/core/basesyntax/dao/ActivityDaoDbImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.storage.Storage;
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ActivityDaoDbImplTest {
+
+    private static ActivityDaoDb activityDaoDb;
+    private static Fruit fruitFirst;
+    private static Fruit fruitSecond;
+
+    @BeforeClass
+    public static void beforeClass() {
+        activityDaoDb = new ActivityDaoDbImpl();
+        fruitFirst = new Fruit("tangerine");
+        fruitSecond = new Fruit("peach");
+    }
+
+    @Test
+    public void test_put_ok() {
+        activityDaoDb.put(fruitFirst, 11);
+        int actualCount = Storage.data.get(fruitFirst);
+        Assert.assertEquals("Must be present Tangerine", 11, actualCount);
```

Comment:
```suggestion
        Assert.assertEquals("Must be present Tangerine", expected, actual);
```
use expected actuals in every test case

Reply:
ok